### PR TITLE
Feature/update requirements with hashes

### PR DESCRIPTION
**Pull request by gemmai95**
Update requirements to include hashes this will help prevent accidental and malicious changes.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[Turn on PIP hashed requirements.txt (3)](https://trello.com/c/ZBaaoggQ/246-turn-on-pip-hashed-requirementstxt-3)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## What? and Why?
-> What does this pull request change/fix? Why was it nessessary?
+> What does this pull request change/fix? Why was it necessary?
```
Comment (necrophonic) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this wrong in the actual template in other repos? If so then worth raising tech debt to fix it

Comment (gemmai95) [inline .github/PULL_REQUEST_TEMPLATE.md]: In the majority I have fixed it as I go along as I have put PRs in on most repo's

Comment (necrophonic) [inline .github/PULL_REQUEST_TEMPLATE.md]: Cool thought you might :)

Comment (gemmai95) [inline .github/PULL_REQUEST_TEMPLATE.md]: According to Adam all the SDX repo's have the template so it will likely need a card

**Pull request merged**